### DISCUSSION TITLE
Umstieg von CCU3

### DISCUSSION THREAD

**Discussion by blackbasket** [OpenCCU/OpenCCU#1125] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen,

ich möchte von einer CCU3 auf RM umstellen. Da ich aber eine neue SD-Karte verwenden möchte, habe ich eine Verständnisfrage.

Ich denke, ich habe es korrekt verstanden, dass ich in dem Fall das RPI3-Image verwende. Das CCU3-Image sollte ja nur für ein Update verwendet werden.

Wie verhält es sich aber mit dem Backup? Muss ich, gemäß CCU3-Anleitung, zwei Backups erstellen oder kann ich in dem Fall direkt ein vollständiges Backup zurückspielen?

LG,
Marcel

**Comment by jens-maus** [comment, marked as answer]
> Ich denke, ich habe es korrekt verstanden, dass ich in dem Fall das RPI3-Image verwende. Das CCU3-Image sollte ja nur für ein Update verwendet werden.

Nein, das ist umgedreht: Das `RaspberryMatic-3.XX.YY.YYYYMMDD-ccu3.tgz` für das einmalige Umstellen von CCU3 -> RaspberryMatic nutzen (einfach via WebUI wie ein Update hochladen) und dann für weitere Updates einfach das `RaspberryMatic-3.XX.YY.YYYYMMDD-rpi3.zip` in Zukunft nutzen wenn man eine CCU3 einsetzt (da diese auf einem RaspberryPi3 basiert).

> Wie verhält es sich aber mit dem Backup? Muss ich, gemäß CCU3-Anleitung, zwei Backups erstellen oder kann ich in dem Fall direkt ein vollständiges Backup zurückspielen?

Na im Grunde musst du bei einem Umstieg von einer bereits lauffähigen CCU3 zu RaspberryMatic nur 1x ein Backup anlegen, nämlich einfach als Sicherung falls etwas beim Umstieg schiefgeht und du dann zurückgehen willst irgendwann. Denn beim Umstieg von einer CCU3 zu RaspberryMatic sollte nach einspielen des `*-ccu3.tgz` sofort alles wie gehabt funktionieren ohne das du danach ein Backup zurückspielen musst. Wichtig ist beim Umstieg nur, dass man sicherstellt, dass man von einer CCU3 3.XX.YY exakt auf eine RaspberryMatic 3.XX.YY oder neuer umsteigt. D.h. man darf z.B. nicht von einer CCU3 3.55.10 zu RaspberryMatic 3.55.5 umsteigen weil 5 < 10 ist. Aber im Grunde sollte das alles schon die offizielle Anleitung entsprechend hergeben:

https://github.com/jens-maus/RaspberryMatic/wiki/Installation#umstieg-von-ccu3

**Reply by blackbasket** [reply]
Hallo Jens,

danke für die prompte Antwort. Eine Nachfrage habe ich dennoch ;)

Ich möchte ja bewusst nicht die aktuelle SD-Karte nutzen, sondern diese zur Seite legen. Dann komme ich doch mit dem "Patch" für die CCU3 nicht weiter, oder?

LG,
Marcel

**Reply by IT-VBFK** [reply]
In so einem Fall startest du ja eh von "0". Dh du beginnst mit dem flashen eines RPI3-Images auf eine neue SD-Karte und spielst dann das CCU3-Backup ein. Und: du musst in so einem Fall nicht zuerst alle Add-Ons deinstallieren (weil: selbe Basis)

**Reply by blackbasket** [reply]
Danke :)

Hab's dann auch gestern so gemacht und nach 26h keine Probleme...

Hätte euch sonst morgen berichtet ;)

LG,
Marcel
